backend/workflow_manager/endpoint/base_connector.py

```diff
@@ -51,6 +51,23 @@ def get_fsspec(
         connector_class: UnstractFileSystem = connector(settings)
         return connector_class.get_fsspec_fs()
 
+    def get_fs_connector(
+        self, settings: dict[str, Any], connector_id: str
+    ) -> UnstractFileSystem:
+        """Get an fs connector based specified connector settings.
+
+        Parameters:
+        - settings (dict): Connector-specific settings.
+        - connector_id (str): Identifier for the desired connector.
+
+        Returns:
+        UnstractFileSystem: An unstract fs connector instance.
+        """
+        if connector_id not in connectors:
+            raise ValueError(f"Invalid connector_id: {connector_id}")
```

Comment:
```suggestion
            raise ValueError(f"Connector '{connector_id}' is not supported.")
```
I'm assuming this would be raised only when we decide to remove support for a connector

Reply:
done